README.md

```diff
@@ -3,18 +3,24 @@
 - [Guide to make C++ available as a web application](#guide-to-make-c-available-as-a-web-application)
   - [JSON schema](#json-schema)
   - [CGI script](#cgi-script)
+- [this Apache2 configuration snippet is stored as apache2/apache2.conf](#this-apache2-configuration-snippet-is-stored-as-apache2apache2conf)
   - [Web framework](#web-framework)
   - [Python](#python)
     - [Accessing C++ function from Python](#accessing-c-function-from-python)
     - [Web application](#web-application)
-    - [Long running tasks](#long-running-tasks)
+- [this Python code snippet is later referred to as <<py-calculate>>](#this-python-code-snippet-is-later-referred-to-as-py-calculate)
+- [this Python snippet is stored as src/py/webapp.py](#this-python-snippet-is-stored-as-srcpywebapppy)
+    - [Long-running tasks](#long-running-tasks)
+- [this Python code snippet is later referred to as <<celery-config>>](#this-python-code-snippet-is-later-referred-to-as-celery-config)
+- [this Python code snippet is later referred to as <<py-submit>>](#this-python-code-snippet-is-later-referred-to-as-py-submit)
+- [this Python snippet is stored as src/py/webapp-celery.py](#this-python-snippet-is-stored-as-srcpywebapp-celerypy)
     - [Web service](#web-service)
-  - [JavaScript](#JavaScript)
-    - [Accessing C++ function from JavaScript in web browser](#accessing-c-function-from-JavaScript-in-web-browser)
-    - [Executing long running methods in JavaScript](#executing-long-running-methods-in-JavaScript)
+- [this yaml snippet is stored as src/py/openapi.yaml](#this-yaml-snippet-is-stored-as-srcpyopenapiyaml)
+  - [JavaScript](#javascript)
+    - [Accessing C++ function from JavaScript in web browser](#accessing-c-function-from-javascript-in-web-browser)
```

Comment:
Your toc generator is parsing chapters from codeblocks. This is not right, please undo the toc updates.

Reply:
It was one of the VSCode extensions causing the issue. Fixed now.